expression/builtin_time_test.go

```diff
@@ -1248,3 +1249,27 @@ func (s *testEvaluatorSuite) TestTimeToSec(c *C) {
 		c.Assert(result.GetInt64(), Equals, test.expect)
 	}
 }
+
+func (s *testEvaluatorSuite) TestSecToTime(c *C) {
+	// Test cases from https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_sec-to-time
+	tests := []struct {
+		param  interface{}
+		expect string
+	}{
+		{2378, "00:39:38"},
+		{3864000, "838:59:59"},
+		{-3864000, "-838:59:59"},
+		{86401.4, "24:00:01"},
+		{86401.5, "24:00:02"},
```

Comment:
ok,I'll add the cases.
But the result shows like this, when I use "select sec_to_time(23.5);"
mysql> select sec_to_time(23.5
);
+-------------------+
| sec_to_time(23.5) |
+-------------------+
| 00:00:24          |
+-------------------+
1 row in set
the result doesn't has faction number.
Is my mysql version too old?